packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "3d4a8d7caec7cd0239dbc40e39c8ec4d965a9a67",
-    "ref_origin" : "dcos-mesos-1.0.3-rc1"
+    "ref": "c7b7ca12e9a5d5a2bed92994f534868bda5c1bc9",
+    "ref_origin" : "dcos-mesos-1.0.x-17ac024"
```

Comment:
Please create and use a new dcos-mesos branch named after the latest 1.0.x SHA included in this branch (`7341f9b`), since `17ac024` is a dozen commits behind HEAD of 1.0.x, and `dcos-mesos-1.0.x-17ac024` is currently based on the newer 1.0.x branch (plus dcos-specific cherry-picks)